src/main/java/com/sleekbyte/tailor/format/XcodeFormatter.java

```diff
@@ -74,6 +71,27 @@ public void displayParseErrorMessage() throws IOException {
         System.out.println(inputFile + Messages.COULD_NOT_BE_PARSED);
     }
 
+    @Override
```

Comment:
Shouldn't `getExitStatus()` be implemented in the base class?  It is repeated in both `XcodeFormatter` and `JsonFormatter`.